registrations/test_tasks.py

```diff
@@ -19,6 +19,7 @@
 from registrations.signals import psh_validate_subscribe
 from registrations.tasks import (
     _create_rapidpro_clinic_registration,
+    create_rapidpro_clinic_registration,
```

Comment:
what's the difference between this and the one above?

Reply:
the `_` is a private task that shouldn't be called directly, that's the task that creates the registration. The other task is the public one that should be called directly, that task fetches the identities, updates the identities, and then runs the `_` task.

Reply:
ahh, I see